File: Code/Components/DanpheEMR.ServerModel/AccountingModels/PrimaryGroupModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.AccountingModels
{
    public class PrimaryGroupModel
    {
        [Key]
        public int PrimaryGroupId { get; set; }
        public string PrimaryGroupCode { get; set; }
        public string PrimaryGroupName { get; set; }
        public bool? IsActive { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public int? ModifiedBy { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/accounting/settings/shared/primary-group.model.ts
import * as moment from "moment";
import { ENUM_DateTimeFormat } from "../../../shared/shared-enums";

export class PrimaryGroupModel {
    public PrimaryGroupId: number = 0;
    public PrimaryGroupCode: string = '';
    public PrimaryGroupName: string = '';
    public IsActive: boolean = false;
    public CreatedOn: string = moment().format(ENUM_DateTimeFormat.Year_Month_Day_Hour_Minute);
    public CreatedBy: number = 0;
    public ModifiedOn: string = '';
    public ModifiedBy: number = 0;
}

